File: StudMap/StudMap.Core/WLAN/Fingerprint.cs
using System.Collections.Generic;

namespace StudMap.Core.WLAN
{
    public class Fingerprint
    {
        public Fingerprint()
        {
            AccessPointScans = new List<AccessPointScan>();
        }

        public int NodeId { get; set; }
        public IEnumerable<AccessPointScan> AccessPointScans { get; set; }
    }
}

File: trunk/StudMap.Client.Core/src/de/whs/studmap/client/core/data/Fingerprint.java
package de.whs.studmap.client.core.data;

import java.util.ArrayList;
import java.util.List;

public class Fingerprint {

	private int NodeId;
	private List<AccessPointScan> AccessPointScans;
	
	public Fingerprint()
	{
		AccessPointScans = new ArrayList<AccessPointScan>();
	}
	
	public int getNodeId() {
		return NodeId;
	}

	public void setNodeId(int nodeId) {
		NodeId = nodeId;
	}

	public List<AccessPointScan> getAccessPointScans() {
		return AccessPointScans;
	}

	public void setAccessPointScans(List<AccessPointScan> accessPointScans) {
		AccessPointScans = accessPointScans;
	}
}
